### PR TITLE
chore(poppler-simple): Upgrade to latest version of poppler-simple

### PR DESCRIPTION
**Pull request by pdufour**
Allows us to use calipers-pdf on node 12
See https://github.com/blackbeam/poppler-simple/issues/29

Note:
- Upgraded packages
- Fixed linting errors automatically

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
 'use strict';
 
-var Promise         = require('bluebird');
-var PopplerDocument = require('poppler-simple').PopplerDocument;
-var utils           = require('./utils');
+const Promise         = require('bluebird'); // eslint-disable-line no-shadow
```
Comment (shraddha1704) [inline lib/index.js:3]: Why do we need to add the eslint rule here?

**Review by shraddha1704 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -101,3 +101,4 @@ Please report any issues in the [core Calipers repository](https://github.com/ca
 # Contribute
 
 The easiest and most helpful way to contribute is to find a file that calipers incorrectly measures, and submit a PR with the file. The tests automatically run against all files in the `test/fixtures` directory, so simply drop it into the appropriate subdirectory, and name it according to its size `<width>x<height>x<pages>.pdf`.
+
```
Comment (mgartner) [inline README.md]: erroneous new line?

**Review by mgartner — APPROVED** [review]

File: lib/index.js (hunk 1 above)
Comment (mgartner) [reply lib/index.js:3]: +1?

Comment (pdufour) [reply lib/index.js:3]: it's overriding the global `Promise` . Which is what we want to do, but an eslint error since it's not usually recommended.

Comment (pdufour) [reply lib/index.js:3]: IMO we shouldn't be using Bluebird at all in which case we can remove that line, but that's a conversation for another time.

**Pull request merged**